### ISSUE TITLE
Users cannot customize path of report via subclassing

### ISSUE THREAD

**Issue by okamayana** [schroepf/TestLab#9]
Hi Tobias, thanks a lot for quickly addressing the last issue (`getFileName()` being private).

I just realized another issue: users cannot change the path of the report file by subclassing `XmlRunListener`.

I think the line that returns `outputFile` in `setInstrumentation()` should be turned into a protected method. Something along the lines of:

```
@Override
public void setInstrumentation(Instrumentation instrumentation) {
    /* other initializations */
    File outputFile = getOutputFile();
    /* other initializations */
}

protected File getOutputFile() {
    /* some default implementation */
}
```

This way, users have full control of where the report file ends up.

If you'd like, I can make a PR to address this. Please let me know what you think!

**Comment by schroepf** [comment]
hi @okamayana,

you're right. It would make sense to allow subclasses to provide their own output file instances if they want to.

Can you please have a look at PR #10 and let me know if that works for you?

**Comment by okamayana** [comment]
I took a look at PR #10, it looks perfect for my use case (and I'm sure for most other use cases as well). 

Thanks for the lightning-fast action. You beat me to it :)

**Comment by schroepf** [comment]
Fixed in PR #10 and Release https://github.com/schroepf/TestLab/releases/tag/0.2.2
